docs/guide/src/privacy.md

```diff
@@ -0,0 +1,31 @@
+# Privacy
+
+This document describes the privacy features of the Penumbra protocol, as well as conditions that must be true for privacy to be preserved.
+
+## Transfers
+
+Transfers do not reveal the asset, amount, sender or recipient identity. Transfers are also unlinkable to each other.
+
+## Swaps
+
+Initiating a swap does not reveal the identity of the swapper or the pre-paid claim fee, but it does reveal the assets and amounts in the swap.
+
+Claiming a swap does not reveal the amounts, asset types or the identity of the claimant.
+
+### Conditions
+
+The above holds true as long as there are at least two swaps in a batch.
+
+## Governance Voting
+
+During a vote, the voting power (amount and asset type of the staked note that's used for voting),
```

Comment:
You could optionally add that the identity of the validator that the voter has delegated to is revealed in a delegator vote. It's implicit given you mention that the asset type of the staked note is revealed, but it's possibly more understandable for end users as the identity of their validator.

Reply:
Good point - added that in with a note. Feel free to reword or Resolve if good.